### ISSUE TITLE
[3.3.5] Auto updater ignores a file

### ISSUE THREAD

**Issue by ShinDarth** [TrinityCore/TrinityCore#15851]
Rev: https://github.com/TrinityCore/TrinityCore/commit/abd4e2e0f1142b8a9edfe31e58bdbca1d4e87c42

The auto updater ignores the file ```2015_10_29_00_characters_335.sql```.

it's currently placed under ```sql/old/3.3.5a/characters/59_2015_11_07```  but it doesn't work even if I manually try to place it under ```sql/updates/characters```.

**Comment by Naios** [comment]
![screenshot_1](https://cloud.githubusercontent.com/assets/1146834/11091602/70bf807c-887c-11e5-8f38-f3dd78db58ae.png)

![screenshot_2](https://cloud.githubusercontent.com/assets/1146834/11091604/735aa0aa-887c-11e5-9509-af7974bd15ff.png)

**Comment by ShinDarth** [comment]
@Naios did you try with a revision before of that update or fresh install?

**Comment by Naios** [comment]
I tried it with my default installation... the update won't be applied on fresh installs since the update is listed in the `updates` table.

**Comment by ShinDarth** [comment]
@Naios I know it is hard to reproduce, and everything works fine on fresh installations, but sometimes there are some glitches.

I found a new bug on a TrinityCore installation, which wasn't updated for a while. I erased world database and replaced with latest TDB release 335.60. When I tried to run it, it gave me a strange error (like it tried to import an old update, which wasn't needed). I solved that problem by manually importing the first update needed (2015_11_07_02_world.sql), and then it worked. Updating of characters and auth also worked fine.

I hope that helps
